common/src/main/scala/explore/components/primereact/EnumDropdownView.scala

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components.primereact
+
+import crystal.react.View
+import eu.timepit.refined.types.string.NonEmptyString
+import japgolly.scalajs.react.*
+import lucuma.core.util.Display
+import lucuma.core.util.Enumerated
+import react.common.*
+import react.primereact.Dropdown
+import react.primereact.SelectItem
+import reactST.primereact.components.{Dropdown => CDropdown}
+
+import scalajs.js
+import scalajs.js.JSConverters.*
+
+object EnumDropdownView:
```

Comment:
Would these be eventually moved to lucuma-ui?

Reply:
Yes. That is the idea. It was just easiest to start with them here.